src/client/preview/client_api.js

```diff
@@ -24,6 +24,10 @@ export default class ClientApi {
   }
 
   storiesOf(kind, m) {
+    if (!kind) {
```

Comment:
Additionally, it might be nice to have this be `!kind && typeof kind !== 'string'` to be a bit stricter